server/app.js

```diff
@@ -8,6 +8,7 @@ const dotenv = require("dotenv");
 
 ///////////////////////////////////////////// Configurations ////////////////////////////////////////
 const User = require('./models/user.model');  // register schema for model User
+const ActivityType = require('./models/ActivityType.model'); // register schema for model Activity Type
```

Comment:
This constant does not seem to be used anywhere in this file, what is its purpose? If no purpose, then get rid of this file. Same applies for User constant

Reply:
Done